spm_tutorials/spm-FaceRepParametric_smdl.json

```diff
@@ -0,0 +1,58 @@
+{
+    "BIDSModelVersion": "1.0.0",
+    "Name": "SPM_FaceRepetitionParametric",
+    "Description": "Look at effect of 2nd order parametric modulation",
+    "Input": {
+        "Include": {
+            "task": "FaceRepetition"
+        }
+    },
+    "Nodes": [
+        {
+            "Level": "subject",
```

Comment:
I am going to try these model specs against the new fitlins refactor.
The first level is required to be the `run` even if there is only one run per subject. You can name the node "subject"
```
"Level": "run",
"Name:" "subject",
```
Example model: https://github.com/poldracklab/fitlins/blob/747c91441f4e34eed31be07683e02af0e2700bfb/examples/models/ds000003/models/model-001_smdl.json

Reply:
I will try to push a fmriprep preprocessed version of those datasets on GIN to help us (and others) for this king of thing.

I might have to coordinate with the SPM team to see if they want to do the hosting themselves somewhere else eventually.

Reply:
That would be awesome. Thanks.

Reply:
Pushed the MoAE data set on GIN

- raw: https://gin.g-node.org/RemiGau/spm_moae_raw
- frmriprep: https://gin.g-node.org/RemiGau/spm_moae_fmriprep

Will work on the face repetition

Reply:
face repetition data set

- raw: https://gin.g-node.org/RemiGau/spm_facerep_raw
- fmriprep derivatives: https://gin.g-node.org/RemiGau/spm_facerep_fmriprep

@shashankbansal6 

here are the datasets

will update the models 

FYI: 
the facerep dataset will most likely be changed in the future to incorporate HED tags in the events.tsv / events.json
this will require some update in the model but we'll burn that bridge when we get to it

Reply:
I ll try to test them this week. Thanks.